### PR TITLE
Client

### PR DESCRIPTION
**Pull request by negreacristian**

### CODE REVIEW

**Comment by negreacristian** [comment]
Cu asa prieteni mai bine dusmani

File: src/client.c [hunk 1]
```diff
@@ -65,3 +65,5 @@ int main()
     close(sock);
     return 0;
 }
+
+// primul comm
```
Comment (ghost) [inline src/client.c]: Sterge si adauga eof line

Comment (ghost) [inline src/client.c]:
```suggestion
// primul comm

```

**Comment by ghost** [comment]
FunWithDickAndJaneFunWithDickAndJaneMovieGIF

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+add_executable(client1 src/client1.cpp)
+add_executable(clienta src/clienta.cpp)
+add_executable(contur src/contur.cpp)
+add_executable(canny src/canny.cpp)
+add_executable(rotate src/rotate.cpp)
+
+# Link OpenCV libraries
+target_link_libraries(contur ${OpenCV_LIBS})
+target_link_libraries(canny ${OpenCV_LIBS})
+target_link_libraries(rotate ${OpenCV_LIBS})
```
Comment (ghost) [inline CMakeLists.txt]:
```suggestion
target_link_libraries(rotate ${OpenCV_LIBS})

```

File: .gitignore [hunk 3]
```diff
@@ -50,3 +50,8 @@ modules.order
 Module.symvers
 Mkfile.old
 dkms.conf
+
+*.log
+
+# Build directory
+build/
```
Comment (ghost) [inline .gitignore]:
```suggestion
build/

```

**Pull request merged**